runtime/bcutil/jimagereader.c

```diff
@@ -117,6 +122,7 @@ j9bcutil_loadJImage(J9PortLibrary *portlib, const char *fileName, J9JImage **pji
 	UDATA fileNameLen = strlen(fileName);
 	IDATA bytesRead = 0;
 	UDATA byteAmount = 0;
+	U_32 mmapflag = OMRPORT_MMAP_FLAG_READ;
```

Comment:
I tried some internal compiles on xlinux and zlinux, which were fine. But I wonder if some platform would fail to compile because of a warning (as error) about an unused variable. Pls #ifdef this variable.

Reply:
`mmapflag` is used at line 194 on all platforms:
`j9mmap_map_file(jimagefd, 0, mapSize, fileName, mmapflag, J9MEM_CATEGORY_CLASSES);`